config/locales/en.yml

```diff
@@ -6,19 +6,51 @@ en:
       friendly: "%e %B %Y at %H:%M"
       blog: "%e %B %Y"
   helpers:
+    label:
+      event:
+        description: "Description"
+        location: "Location"
+        location_url: "Location URL"
+        microcosm_id: "Microcosm"
+        moment: "When"
+        submit: "Create Event2"
```

Comment:
Is the "2" correct?

Reply:
The whole submit is not correct.  The submit field/button doesn't have a <label>.  I have to look into internationalizing the text on the submit button as it behaves differently from other form fields.  Thanks.